pkg/prometheus/bc.go

```diff
@@ -0,0 +1,45 @@
+package prometheus
+
+import (
+	"github.com/anodot/anodot-common/pkg/metrics3"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	log "k8s.io/klog/v2"
+	"time"
+)
+
+func NewPipeline(startTime metrics3.AnodotTimestamp) metrics3.Pipeline {
+	pipeline := metrics3.Pipeline{}
+	pipeline.Id = "prometheus"
+	pipeline.Created = startTime
+	pipeline.Updated = startTime
+	pipeline.Status = "RUNNING"
+
+	s := metrics3.Source{}
+	s.Name = "prometheus"
+	s.Type = "prometheus"
+	pipeline.Source = s
+
+	return pipeline
+}
+
+var sendStatusToBCErrors = promauto.NewCounter(prometheus.CounterOpts{
+	Name: "anodot_send_status_to_bc_errors",
```

Comment:
anodot_send_status_to_bc_errors  - rename to `anodot_send_status_to_bc_errors_count` or `anodot_send_status_to_bc_total` ? 

https://prometheus.io/docs/practices/naming/